### PR TITLE
Support all GitHub themes

### PR DESCRIPTION
**Pull request by ClementGaston**
Hello,

I wanted to use your library, but it was missing a lot of GitHub theme. I decided to edit your work to support all current (and future) themes provided by GitHub (for the markdown). I let you see what I have done.

Here a snippet of my work: 
https://gyazo.com/556bb260ce12f954610a78365800bd11 

If you are happy of my work, I want to add next an option to output minified CSS :)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -28,21 +28,26 @@ $ npm install --global generate-github-markdown-css
...
+	  --version   	  Show version.
+
+	Examples
+	  $ github-markdown-css --list
+	  light
+	  dark
+	  dark_dimmed
+	  dark_high_contrast
+	  dark_colorblind
+	  light_colorblind
```
Comment (sindresorhus) [inline readme.md:50]: This is not part of the actual CLI output

File: test.js [hunk 2]
```diff
@@ -5,9 +5,24 @@ import githubMarkdownCss from './index.js';
...
+	const themes = (await githubMarkdownCss({list: true})).split('\n');
+
+	fs.writeFileSync('dist/all.css', await githubMarkdownCss());
+
+	const promises = [];
+
+	for (const theme of themes) {
+		promises.push(new Promise((resolve, reject) => {
+			githubMarkdownCss({themes: [theme]})
+				.then(css => {
```
Comment (sindresorhus) [inline test.js:17]: Use Async/await.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
I think `auto` should still be there for users that just want light/dark and auto-switching.

**Comment by sindresorhus** [comment]
Nice work :)

**Comment by DarrenDanielDay** [comment]
Looking forward to this feature for `dark dimmed` theme 👀

**Pull request closed without merging**